# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/no-fc-cache-run.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

The run of fc-cache makes xfree86 depend on any other font package - which
is -well- sub-optimal ... ;-)

  - Rene Rebe <rene@rocklinux.org>

--- xc/programs/fc-cache/Imakefile.orig	2003-09-19 00:31:32.000000000 +0200
+++ xc/programs/fc-cache/Imakefile	2003-09-19 00:26:02.000000000 +0200
@@ -15,7 +15,7 @@
 
 install::
 	if [ x$${DESTDIR} = x ]; then \
-		$(FCCACHE) -v -f;\
+		echo "You should run $(FCCACHE) -v -f later ..." ;\
 	fi
 
 LinkSourceFile(fc-cache.c,$(FONTCONFIGDIR)/fc-cache)
--- xc/config/cf/X11.tmpl.orig	2003-09-09 20:05:52.000000000 +0300
+++ xc/config/cf/X11.tmpl	2003-10-18 15:18:04.000000000 +0300
@@ -3523,9 +3523,7 @@
 # endif /* MakeFcCache */
 
 # ifndef InstallFcCache
-#  define InstallFcCache(dest)						@@\
-install::								@@\
-	RunProgram(FCCACHE, $(FCCACHEOPTS) $(DESTDIR)dest)
+#  define InstallFcCache(dest)
 # endif /* InstallFcCache */
 
 #else
